### PR TITLE
made the version compatible with EF 7 changes and minor changes to make the Test run

### PR DESCRIPTION
**Pull request by BilalHasanKhan**
I have tried my bit, this is my first PR in case I have not followed any convention or rule please be kind and do let me know and I shall make the ammendment.

### CODE REVIEW

File: src/OrchardVNext.Web/App_Data/Sites/Default/settings.txt [hunk 1]
```diff
@@ -2,8 +2,8 @@ State: Running
 Themes: 
 Modules: 
 Name: Default
-DataProvider: SqlCe
-DataConnectionString: null
+DataProvider: SqlServer
+DataConnectionString: Data Source=.\SQLExpress;Initial Catalog=OrchardvNext;Persist Security Info=True;Integrated Security=SSPI;
```
Comment (Jetski5822) [inline src/OrchardVNext.Web/App_Data/Sites/Default/settings.txt]: Could you revert this change

**Comment by Jetski5822** [comment]
Awesome changes!! and thanks. If you make that one change, I will merge :+1:

**Comment by BilalHasanKhan** [comment]
Reverted the file let me know if you see it that way

**Comment by Jetski5822** [comment]
Brill thanks :)

**Comment by BilalHasanKhan** [comment]
Thanks :) ... In case I would like to get more involved specifically in vNext is there any official roadmap.
From the weekly podcast it seems there is none yet.
Could you please let me know any specific channels to hear the chatter for vNext

**Comment by Jetski5822** [comment]
@BilalHasanKhan Sure, here is the roadmap to start with https://github.com/OrchardCMS/Brochard/wiki/Roadmap
